modules/mesh-layers/src/simple-mesh-layer/simple-mesh-layer-vertex.glsl.js

```diff
@@ -27,11 +28,14 @@ out vec4 position_commonspace;
 out vec4 vColor;
 
 void main(void) {
+  // https://github.com/Esri/i3s-spec/blob/master/docs/1.7/geometryUVRegion.cmn.md
```

Comment:
This layer is not an I3S layer. Why is there a reference of I3S specification?

Reply:
Ok. As soon as this is I3S - specific feature, I will implement it in `_MeshLayer`

Reply:
I feel this can be presented as a generic feature. Basically, this just allows different vertices to use different sub textures in a single atlas texture in a single material. Just write some generic documentation for the feature, rather than mentioning I3S.